ignite/src/test/java/org/apache/zeppelin/ignite/IgniteSqlInterpreterTest.java

```diff
@@ -58,14 +58,14 @@ public void setUp() {
 
     IgniteConfiguration cfg = new IgniteConfiguration();
     cfg.setDiscoverySpi(discoSpi);
+    cfg.setPeerClassLoadingEnabled(true);
```

Comment:
default-ignite-jdbc.xml needs for JDBC connection that based on Ignite client node. But from test code we start Ignite server node with different configuration.